PLASTIC-19024 add group api documentation

jasonyzw opened the pull request:
## Context
Adding group related endpoints into a `groups-api` collection to make group functionality more discoverable and organized

## Checklist
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

jasonyzw committed:
added groups api

jasonyzw committed:
updated README

timotheehub commented on groups-api/Groups API Sandbox (empty).postman_environment.json:
```diff
@@ -0,0 +1,33 @@
+			"enabled": true
+		},
+		{
+			"key": "profile-id",
```
Let's call it `profile_id` to be consistent with the other Postman collection.

jasonyzw replied:
Changed!

timotheehub commented on groups-api/Groups API Sandbox (empty).postman_environment.json:
```diff
@@ -0,0 +1,33 @@
+			"enabled": true
+		},
+		{
+			"key": "group-id",
```
Same here. Let's call it `group_id`.

jasonyzw replied:
Changed!

timotheehub commented on groups-api/Groups API Sandbox (empty).postman_environment.json:
```diff
@@ -0,0 +1,33 @@
+			"type": "default",
+			"enabled": true
+		},
+		{
```
We can also add the refresh token so that we can add the `POST {{host}}/oauth/token` API endpoint.

jasonyzw replied:
Added both refersh and token endpoint for ease of test!

jasonyzw committed:
update according to comments

jasonyzw commented:
@timotheehub I have addressed the comments, but "Create balance group" is still a problem when I try to test on Sandbox.
Maybe we can re-review and get it approved (if there is no more enhancement).
But not merge until we resolve the endpoint issue

jasonyzw commented:
@timotheehub have retested on Sandbox and it seems to be working now! 
The PR is ready for re-review!

timotheehub commented on groups-api/Groups API.postman_collection.json:
```diff
@@ -0,0 +1,550 @@
+								},
+								{
+									"key": "refresh_token",
+									"value": "{{refresh_token}}",
```
I believe we use `refresh` everywhere else.

timotheehub reviewed (commented):
We'll need to solve the issue with the feature flag. It's fine to merge first though. We could skip the group feature flag if the token is from the partner.
